fastlane/Fastfile

```diff
@@ -154,13 +154,18 @@ platform :android do
     # This dependency could be removed with a little more time to tidy up this script to do the branching/merging manually.
 
     desc "Create new release"
-    lane :release do
+    lane :release do |option|
 
         ensure_git_status_clean
         ensure_git_branch( branch: 'develop' )
 
-        newVersion = determine_version_number()
-        releaseNotes = determine_release_notes()
+        newVersion = determine_version_number(
+            release_number: options[:release_number]
+        )
+        releaseNotes = determine_release_notes(
+            release_notes: options[:release_notes],
```

Comment:
Can you provide release notes which are multi-line using this mechanism?

Reply:
At least from jenkins, I tried to set mutline params and passed multiline values and it works as expected.

Reply:
For posterity, this is also fine from local command line.

`fastlane release release_notes:"`  would be enough to trigger the multi-line input, and ending with `"` would allow the multi-line release notes to be added.

Thanks for the suggestion @karlenDimla.